gcsweb/README.md

```diff
@@ -24,3 +24,11 @@ links to specific files.
 2. `gcsweb` is not designed for initial browsing (yet?) - it doesn't list
 which GCS buckets are available, and because bucket is a part of URL, you
 need a documented link to browse.
+
+#### Installation and deployment
+
+`gcsweb` is built from source code at
+http://github.com/kubernetes/test-infra/gcsweb into Docker container, which
+is then uploaded to private container storage at
+https://gcr.io/google_containers/gcsweb-amd64 and fetched during processing
```

Comment:
Is this repository aims at providing best practices https://github.com/kubernetes/k8s.io/issues/23#issue-182170987 then documenting things like `kubectl apply` is necessary. Reading small paragraph here may save hours of various tutorials elsewhere. I would record video with example session from scratch (given that cluster is already there), but as I said I don't have any cluster to experiments (no credit card), so it is a pure theory for me.